### ISSUE TITLE
Save Email to Files

### ISSUE THREAD

**Issue by wriver4** [nextcloud/mail#9240]
### Is your feature request related to a problem? Please describe.

How do I save and email to a Files folder? The current option is to download it to a device. Then upload it to the folder using DAV folder on device. 

### Describe the solution you'd like

Have a Save to Files feature.

### Describe alternatives you've considered

The current option is to download it to a device. Then upload it to the folder using DAV folder on device.

### Additional context

For those of us who want to have a solid records management system, having the emails stored in the related folder is important.
Consider a lawsuit where the disclosure is requested, emails are an important part of that disclosure. The easiest solution is to have the user save the emails to that topic in files.

**Comment by ChristophWurst** [comment]
Duplicate of https://github.com/nextcloud/mail/issues/675?

**Comment by wriver4** [comment]
No, not automatically, manually and selectively. Example, sales has an email discussion with client about specifications attachments back and forth, and they approve the last one by email with no attachment just a reference#, I want to save that email in Files - client folder. Same scenario with a service contract #.

**Comment by wriver4** [comment]
Also, multiple departments and people within the departments interact with the presales process, having one place to dump all the related emails in case the client said you never told me that would be helpful.

**Comment by the-djmaze** [comment]
Install SnappyMail app.
Do the steps you want.
Make screenshots of the process.
Post that here.

This way they know how to implement what you want (i'm not gonna do this for you 😉 )

**Comment by wriver4** [comment]
@the-djmaze How does installing SnappyMail app improve Nextcloud Mail?

**Comment by the-djmaze** [comment]
Not. And that is not what i'm saying.
I say SnappyMail can what you want:

![afbeelding](https://github.com/nextcloud/mail/assets/3752035/a757ff0a-eb33-4344-bcc3-633bfc94400c)
![afbeelding](https://github.com/nextcloud/mail/assets/3752035/0bbb4751-6474-4926-bd2f-0d6a41d7d567)

**Comment by EpicScope** [comment]
It would be fantastic to have that feature. It would also help transition away from external email servers. People who are hitting the free quota of reputable email servers AND have a Nextcloud installation may benefit from this. I know I would :)
What are the chances this makes it into a future version of the vanilla NC Mail app? Is it significantly maintained at all?
